explorer/graphql/app/schema.graphql

```diff
@@ -66,6 +66,15 @@ type XMsg {
   "Hash of the source chain block"
   BlockHash: Bytes32!
 
+  "Hash of destination chain receipt"
+  ReceiptHash: Bytes32
```

Comment:
What is this exactly? Just hash of the xreceipt bytes? tx receipt hash?

Reply:
tx hash of the receipt not the bytes